.travis.yml

```diff
@@ -13,5 +13,5 @@ script:
     - chmod u+x ./official-git/run-tests.sh
     # t0008 tests that are skipped require ! to negate a pattern. (GitHub issue #72)
     # t3000.7 requires git pack-refs
-    - GIT_SKIP_TESTS='t0008.30[5-9] t0008.310 t0008.321 t0008.323 t0008.37[0-9] t0008.38[0-7] t0008.39[1-2] t3000.7' ./official-git/run-tests.sh t0000-basic.sh t0004-unwritable.sh t0007-git-var.sh t0008-ignores.sh t0010-racy-git.sh t0062-revision-walking.sh t0070-fundamental.sh t0081-line-buffer.sh t1000-read-tree-m-3way.sh t1001-read-tree-m-2way.sh t1002-read-tree-m-u-2way.sh t1003-read-tree-prefix.sh t1009-read-tree-new-index.sh t1100-commit-tree-options.sh t1304-default-acl.sh t2100-update-cache-badpath.sh t3000-ls-files-others.sh t3002-ls-files-dashpath.sh t3006-ls-files-long.sh t3020-ls-files-error-unmatch.sh t4113-apply-ending.sh t4123-apply-shrink.sh t5705-clone-2gb.sh t7062-wtstatus-ignorecase.sh t7511-status-index.sh
```

Comment:
Wow, so many new test suites passing. I wonder what the overall numbers look like now. The last time I checked, we were getting an overall pass rate of about 25%.